edx_sga/sga.py

```diff
@@ -705,6 +808,32 @@ def file_storage_path(self, sha1, filename):
         )
         return path
 
+    def get_user(self):
+        """returns user object"""
+        try:
+            user = self.runtime.get_real_user(self.xmodule_runtime.anonymous_student_id)
+        except Exception:  # pylint: disable=broad-except
+            user = None
+
+        return user
+
+    def is_zip_file_available(self, user):
+        """
+        returns True if zip file available.
+
+        Args:
+            user (django.contrib.auth.models.User): user instance
+        """
+        if user:
```

Comment:
Unnecessary check

Reply:
it is better to add check, i now suer is always there but to be safe i am keeping this check